### PR TITLE
Move CHANGES entry for #8167 to 8.8.2 section

### PR DESCRIPTION
**Pull request by JasonGross**
As per https://github.com/coq/coq/pull/8167#issuecomment-416929865

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -158,7 +154,16 @@ Notations
   Note that this might cause incompatibilities if you have, e.g., list_scope
   and vector_scope both open with vector_scope on top, and expect `++` to
   refer to `app`.
-  Solution: wrap `_ ++ _` in `(_ ++ _)%list` (or whichever scope you want). 
+  Solution: wrap `_ ++ _` in `(_ ++ _)%list` (or whichever scope you want).
```
Comment (Zimmi48) [inline CHANGES]: Why does GitHub think this line has changed? :confused:

File: CHANGES [hunk 2]
```diff
@@ -158,7 +154,16 @@ Notations
...
+
+Changes from 8.8.1 to 8.8.2
+===========================
+
+Tools
+
+- The targets `print-pretty-timed`, `print-pretty-timed-diff`, and
```
Comment (Zimmi48) [inline CHANGES]: Note that in the version I backported I reworded to "The coq-makefile targets" to be clearer without context.

**Review by Zimmi48 — APPROVED** [review]
Thanks!

File: CHANGES (hunk 1 above)
Comment (JasonGross) [reply CHANGES]: I auto-deleted a trailing whitespace.

File: CHANGES (hunk 2 above)
Comment (JasonGross) [reply CHANGES]: Thanks, updated

**Pull request merged**